.okhttpcache/5d6d770102281931c035b08c519c42d1.0

```diff
@@ -0,0 +1,37 @@
+https://api.github.com/repos/ghacupha/java_text_tables/issues?state=all&per_page=100&page=1
```

Comment:
This folder, `.okhttpcache` does not appear like it should be added to the repo.

Did you forget to remove it from the PR?

Depending on how it gets created, I might consider adding it to `.gitignore`